ui/cast_button.js

```diff
@@ -94,12 +94,12 @@ shaka.ui.CastButton = class extends shaka.ui.Element {
 
     /** @private */
   async onCastClick_() {
-    if (this.castProxy_.isCasting()) {
-      this.castProxy_.suggestDisconnect();
+    if (this.castProxy_['isCasting']()) {
```

Comment:
I believe these changes to bracket notation can all be reverted once you fix the externs file.